### PR TITLE
homopolish v0.4.1

### PR DESCRIPTION
**Pull request by shelby-bennett**
Hi there! 

I would like to add homopolish v0.4.1 to the StaPH-B dockerbuilds. The Dockerfile successfully builds to test, and everything passed a manual test in github actions (although it did take incredibly long). This is my first solo dockerfile build (shout out to the APHL docker training), so please feel free to let  me know of any areas in need of improvement. 

Included in this pull request: 
homopolish dockerfile
homopolish README.md
updated general README.md
updated licenses 

Pull Request (PR) checklist:
- [ x] Include a description of what is in this pull request in this message.
- [ x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [ x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [ x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [ x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [ x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [ x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ To learn more about the docker pull rate limits and the open source software pro
 | [GAMBIT](https://hub.docker.com/r/staphb/gambit) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/gambit)](https://hub.docker.com/r/staphb/gambit) | <ul><li>0.3.0</li><li>0.4.0</li><li>0.5.0</li><li>1.0.0</li></ul> | https://github.com/jlumpe/gambit |
 | [GAMMA](https://hub.docker.com/r/staphb/gamma) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/gamma)](https://hub.docker.com/r/staphb/gamma) | <ul><li>1.4</li><li>2.1</li><li>2.2</li></ul> | https://github.com/rastanton/GAMMA/ |
 | [hmmer](https://hub.docker.com/r/staphb/hmmer) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/hmmer)](https://hub.docker.com/r/staphb/hmmer) | <ul><li>3.3</li></ul> | http://hmmer.org/ |
+| [homopolish](https://hub.docker.com/r/staphb/homopolish) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/homopolish)](https://hub.docker.com/r/staphb/homopolish) | <ul><li>0.4.1</li></ul> | https://github.com/ythuang0522/homopolish/ |
```
Comment (erinyoung) [inline README.md:77]: This looks great!

At the bottom of this file, we have everyone link their profile. Could you add yours, too?

**Comment by erinyoung** [comment]
This looks great!

I only request that you add your github tag to the bottom of the main README.

File: README.md (hunk 1 above)
Comment (shelby-bennett) [reply README.md:77]: Hey Erin, 

I just checked and it appears that my github is already added to the file(line 223).

**Comment by erinyoung** [comment]
I retract my statement. Sorry! I'm going to merge this and push to Dockerhub and Quay. Let me know if you have issues.

**Review by erinyoung — APPROVED** [review]

**Pull request merged**